### ISSUE TITLE
[Feature Request] Associating files without icon change

### ISSUE THREAD

**Issue by ahmetsait** [notepad-plus-plus/notepad-plus-plus#2209]
Currently, when a file type is associated with Notepad++ the icon setting for the file type is overwritten so that auto generated file icon (paper with notepad++ icon on it) shown in explorer. This is not the thing I would want most of the time, since I want the icon Visual Studio have put for .cpp files but obviously I don't want to open a single cpp file in VS, so I configure it to notepad++.
So basically, if Notepad++ had an _alternative_ option for associating files without altering icons, it would be awesome.

**Comment by MAPJe71** [comment]
AFAIK it's standard Windows behavior that a file shows an icon of the application with which it's associated. This shows the user which application will be used to open it on double click by default. Use right-click-Open-with to incidentally select a different application.

All that said, it might be helpful to have type specific icons e.g. Notepad++ icon with a C++ overlay.
